.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+repos:
```

Comment:
I'm thinking the removal of `.bandit` and `.mypy.ini` is unrelated to the precommit file, is this correct?

Reply:
`pre-commit` brings together these configurations in a single file. Then the code base can be check by simply running `pre-commit run --all-files`. The bandit configuration is comparable, but `mypy.ini` indeed was a bit stricter than the current `pre-commit` config.